compiler/tests-common-new/tests-gen/org/jetbrains/kotlin/test/runners/ForeignAnnotationsSourceJavaTestGenerated.java

```diff
@@ -98,6 +98,30 @@ public void testIrrelevantQualifierNicknames() {
       runTest("compiler/testData/diagnostics/foreignAnnotationsTests/tests/irrelevantQualifierNicknames.kt");
     }
 
+    @Test
```

Comment:
I don't quite understand why only this class fails the test. I would like to ask for help.

Reply:
thanks, I will look into it a bit more myself.

Reply:
fixed test